libkmod/libkmod-index.c

```diff
@@ -167,24 +167,16 @@ static int add_value(struct index_value **values, const char *value, size_t len,
 	return 0;
 }
 
-static int read_char(FILE *in)
+static inline int read_char(FILE *in)
```

Comment:
There is no need for the `inline` here - the compiler will do it as applicable.

Reply:
```
$ grep -E '^static .+ read_' libkmod/libkmod-index.c
static inline int read_char(FILE *in)
static bool read_u32s(FILE *in, uint32_t *l, size_t n)
static inline bool read_u32(FILE *in, uint32_t *l)
static inline uint32_t read_u32_mm(const void **p)
static inline uint8_t read_char_mm(const void **p)
static inline void read_value_mm(const void **p, struct index_mm_value *v)
```
adding inline there make all the `read_*` more consistent. Anyway, as you said, the compiler will do what's applicable.